### DISCUSSION TITLE
Is the process of creating a AGI under controlled? Do people really know what they are creating and how they are creating it? Should it be controlled, or, it's not that time now?

### DISCUSSION THREAD

**Discussion by lets-make-safe-ai** [lets-make-safe-ai/make-safe-ai#18] [General]
Is the process of creating a AGI under controlled? Do people really know what they are creating and how they are creating it? Should it be controlled, or, it's not that time now?

**Comment by chirag127** [comment]
yes, people know, you are just afraid of the ai shown in sci fi movies
